src/CLI/clitree/cli-xml/interface.xml

```diff
@@ -22,6 +22,12 @@ limitations under the License.
     xsi:schemaLocation="http://www.dellemc.com/sonic/XMLSchema
     http://www.dellemc.com/sonic/XMLSchema/clish.xsd"
     >
+    <PTYPE
+    name="MODE"
+    method="select"
+    pattern="active on"
+    help="PortChannel Mode"/>
+
```

Comment:
Can we move this PTYPE to sonic_types.xml.

Reply:
Addressed